bgpd/bgp_zebra.c

```diff
@@ -2114,6 +2114,24 @@ int bgp_redistribute_unset(struct bgp *bgp, afi_t afi, int type,
 	return CMD_SUCCESS;
 }
 
+int bgp_redistribute_unset(struct bgp *bgp, afi_t afi, int type,
+			   unsigned short instance)
+{
+	struct listnode *node, *nnode;
+	struct bgp_redist *red;
+
+	if (type != ZEBRA_ROUTE_TABLE || instance != 0)
+		return _bgp_redistribute_unset(bgp, afi, type, instance);
```

Comment:
_bgp_redistribute_unset() probably could be a void at all. We always return CMD_SUCCESS inside, and here too.

Reply:
done